src/Symfony/Bundle/FrameworkBundle/Resources/views/Form/choice_widget.html.php

```diff
@@ -1,5 +1,17 @@
+<?php if (false !== isset($empty_choices)): ?>
 <?php if ($expanded): ?>
 <?php echo $view['form']->block($form, 'choice_widget_expanded') ?>
 <?php else: ?>
 <?php echo $view['form']->block($form, 'choice_widget_collapsed') ?>
 <?php endif ?>
+<?php else: ?>
+<?php if ('' === $empty_choices) {
```

Comment:
right, the condition on the first line is wrong, it should be :
` +<?php if (!isset($empty_choices)): ?>`